libsolidity/codegen/ir/IRGeneratorForStatements.h

```diff
@@ -86,10 +86,19 @@ class IRGeneratorForStatements: public IRGeneratorForStatementsBase
 	IRVariable evaluateExpression(Expression const& _expression, Type const& _to);
 
 	/// Defines @a _var using the value of @a _value while performing type conversions, if required.
-	/// If @a _forceCleanup is set to true, it also cleans the value of the variable after the conversion.
-	void define(IRVariable const& _var, IRVariable const& _value, bool _forceCleanup = false)
+	void define(IRVariable const& _var, IRVariable const& _value)
 	{
-		declareAssign(_var, _value, true, _forceCleanup);
+		bool _declare = true;
+		declareAssign(_var, _value, _declare);
+	}
+
+	/// Defines @a _var using the value of @a _value while performing type conversions, if required.
+	/// It also cleans the value of the variable after the conversion.
```

Comment:
Is "cleaning up after the conversion" really what happens here? In a sense it is... but I'd rather say, it performs a cleanup, even if ``@a _var`` and ``@a _value`` have the same type. Similarly for ``convertAndCleanup``